geometry_msgs/PoseStamped[] waypoints
std_msgs/Float64[] velocities